cmd/plural/cd_prautomation.go

```diff
@@ -0,0 +1,63 @@
+package plural
+
+import (
+	"io"
+	"os"
+
+	"github.com/pluralsh/plural-cli/pkg/utils"
+	"github.com/samber/lo"
+	"github.com/urfave/cli"
+)
+
+func (p *Plural) cdPrAutomations() cli.Command {
+	return cli.Command{
+		Name:        "pr-automation",
```

Comment:
I think this should be within the `pr` subcommand (I think there's already one).  That way `plural pr create ...` reads a lot more clearly

Reply:
done